### PR TITLE
fix #2819

### LINKED ISSUE #2819: zig fmt: fix whitespace
* [ ] tabs -> spaces
 * [ ] CRLF -> LF

Tabs are not up for debate in this issue, you can post your comments in #544. Please respect the fact that many people have clicked "unsubscribe" on that issue, by exclusively using #544 to complain about tabs.

### PR DESCRIPTION
**Pull request by emekoi**
this closes #2819 and formats the stdlib.

### CODE REVIEW

File: std/special/compiler_rt/comparetf2.zig [hunk 1]
```diff
@@ -74,14 +74,12 @@ pub extern fn __getf2(a: f128, b: f128) c_int {
...
-            GE_EQUAL
-        else
-            GE_GREATER
-    else if (aInt > bInt)
+    return if ((aInt & bInt) >= 0) if (aInt < bInt)
+        GE_LESS
+    else if (aInt == bInt)
+        GE_EQUAL
+    else
+        GE_GREATER else if (aInt > bInt)
```
Comment (fengb) [inline std/special/compiler_rt/comparetf2.zig]: I think this should be refactored.

Comment (emekoi) [reply std/special/compiler_rt/comparetf2.zig]: the formatting or the actual code?

Comment (fengb) [reply std/special/compiler_rt/comparetf2.zig]: https://github.com/ziglang/zig/issues/2661

It's not really related to your change but trying to parse this hurts my brain.

Comment (emekoi) [reply std/special/compiler_rt/comparetf2.zig]: i'll put some `//zig fmt: off`/`//zig fmt: on` blocks around it.

**Comment by andrewrk** [comment]
This doesn't print error messages correctly:

```zig
const std = @import("std");

pub fn main() void {
	std.debug.warn("subsystem = {}\n" std.os.windows.subsystem);
}
```

```
$ zig fmt test.zig 
test.zig:4:36: error: expected RParen, found Identifier
	std.debug.warn("subsystem = {}\n" std.os.windows.subsystem);
                                   ~~~
```

The squiggles should be pointing at `std`.

**Pull request closed without merging**